### DISCUSSION TITLE
CEFSharp Rendering Issue

### DISCUSSION THREAD

**Discussion by TonyValenti** [cefsharp/CefSharp#3621] [Q&A]
Hi @amaitland -
I was upgrading CEF Sharp to the latest version hoping that it would resolve this issue:
https://www.youtube.com/watch?v=OlYb7GFYUfo

Short story -
On some computers, the control only redraws when the form is resized.  I have not been able to narrow this down to a specific platform.  Also, Chrome and Edge seem to work fine on the PCs.

Any ideas?

**Comment by amaitland** [comment]
Are you using WinForms or WPF? 

WPF there is a known issue with Intel 11th Gen iris graphics cards. 

- https://www.intel.com/content/www/us/en/support/articles/000058136/graphics/graphics-for-11th-generation-intel-processors.html
- https://github.com/dotnet/wpf/issues/3817

**Reply by PaulParau** [reply]
We have encountered this exact same error: we are using WPF and a few of our clients are having this issue and we have been unable to reproduce it on our machines. We will recommend that they update their graphics drivers.
